docs/api/cpp/color_space.rst.in

```diff
@@ -0,0 +1,30 @@
+..
```

Comment:
[Question] Do we need to create a new xxx.rst.in file anytime we create a new class xx ?